llama_stack/distribution/datatypes.py

```diff
@@ -234,6 +234,17 @@ class AuthenticationConfig(BaseModel):
     )
 
 
+class QuotaSqliteConfig(BaseModel):
+    db_path: str = Field(default="./quotas.db", description="Path to the SQLite DB file")
+    quota_requests_per_day: int = Field(default=1000, description="Maximum requests per client per window")
```

Comment:
Should we control per day? or let's the admin decide? per hour?

Reply:
We should control per day; per hour is too granular, in my opinion.